build-javascript/build.sh

```diff
@@ -5,8 +5,8 @@ set -e
 # Config
 
 export SCONS="scons -j${NUM_CORES} verbose=yes warnings=no progress=no"
-export OPTIONS="production=yes"
-export OPTIONS_MONO="module_mono_enabled=yes mono_static=yes mono_prefix=/root/mono-installs/wasm-runtime-release use_lto=no"
+export OPTIONS="production=yes use_thinlto=yes"
```

Comment:
@Faless We'd have to do a proper benchmark but I think using ThinLTO for the Web platform would be better than (full) LTO. Clang/Emscripten's full LTO is single thread and just takes hours despite running on a ThreadRipper. While ThinLTO is multithreaded and takes minutes.

Reply:
As seen in #66 this isn't the best option in the end so I'm reverting this change in #67.